projects/sbb-esta/angular-business/chip/src/chip/chip.component.ts

```diff
@@ -0,0 +1,37 @@
+import { Component, EventEmitter, HostBinding, Input, Output } from '@angular/core';
+
+@Component({
+  selector: 'sbb-chip',
+  templateUrl: './chip.component.html',
+  styleUrls: ['./chip.component.scss']
+})
+export class ChipComponent {
+  @Input()
+  label: string;
+
+  @Input()
+  disabled = false;
+
+  @Output()
+  dismissed: EventEmitter<boolean> = new EventEmitter();
+
+  @HostBinding('hidden') hidden: 'false' | 'true';
```

Comment:
Sorry, that was my mistake. I meant this to be a boolean with default `false`.

Reply:
Ah I think I got it now. I reverted back to setting `ariaHidden` first, as it seems to be the common implementation practice, and added the `null | undefined` types. It seems to avoid rendering anything in the DOM.

Reply:
I'm an idiot. I meant `null | boolean`. 😢 
Ok, that's fine and good to know.

Reply:
Done :)